src/index.html

```diff
@@ -11,6 +11,48 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+
+    <div class="conteiner">
+      <form 
+        class="search__form"
+        data-qa="big"
+        action="/"
+        methot="get">
```

Comment:
be attentive with typos
```suggestion
        method="get">
```